src/openfl/_internal/renderer/opengl/batcher/BatchRenderer.hx

```diff
@@ -371,14 +375,27 @@ class BatchRenderer {
 				lastBlendMode.apply(gl);
 			}
 
+			// set vertex attribute pointers to the start of the group inside the vertex data
+			gl.vertexAttribPointer(shader.aX,                  4, gl.FLOAT, false, stride, offset);
+			gl.vertexAttribPointer(shader.aY,                  4, gl.FLOAT, false, stride, offset + 4 * Float32Array.BYTES_PER_ELEMENT);
+			gl.vertexAttribPointer(shader.aU,                  4, gl.FLOAT, false, stride, offset + 8 * Float32Array.BYTES_PER_ELEMENT);
+			gl.vertexAttribPointer(shader.aV,                  4, gl.FLOAT, false, stride, offset + 12 * Float32Array.BYTES_PER_ELEMENT);
+			gl.vertexAttribPointer(shader.aTextureId,          1, gl.FLOAT, false, stride, offset + 16 * Float32Array.BYTES_PER_ELEMENT);
+			gl.vertexAttribPointer(shader.aColorOffset,        4, gl.FLOAT, false, stride, offset + 17 * Float32Array.BYTES_PER_ELEMENT);
+			gl.vertexAttribPointer(shader.aColorMultiplier,    4, gl.FLOAT, false, stride, offset + 21 * Float32Array.BYTES_PER_ELEMENT);
+			gl.vertexAttribPointer(shader.aPremultipliedAlpha, 1, gl.FLOAT, false, stride, offset + 25 * Float32Array.BYTES_PER_ELEMENT);
 
 			// draw this group's slice of vertices
-			gl.drawElements(gl.TRIANGLES, group.size * 6, gl.UNSIGNED_SHORT, group.start * 6 * UInt16Array.BYTES_PER_ELEMENT);
+			instancedRendering.drawArraysInstanced(gl.TRIANGLE_STRIP, 0, 4, group.size);
```

Comment:
Can we somehow avoid setting `vertexAttribPointer` offsets per group by using the `first` argument of `drawArraysInstanced` properly?